plugin/al/lib/AL_USDMaya/AL/usdmaya/nodes/ProxyShape.cpp

```diff
@@ -1905,23 +1905,6 @@ MSelectionMask ProxyShape::getShapeSelectionMask() const
     return MSelectionMask(selType);
 }
 
-#if defined(WANT_UFE_BUILD)
-//----------------------------------------------------------------------------------------------------------------------
-Ufe::PathSegment ProxyShape::ufePathSegment() const
-{
-    // Build a path segment to proxyShape
-    MDagPath thisPath;
-    MDagPath::getAPathTo(thisMObject(), thisPath);
-
-    // MDagPath does not include |world to its full path naem
-    MString fullpath = "|world" + thisPath.fullPathName();
-
-    return Ufe::PathSegment(fullpath.asChar(), MAYA_UFE_RUNTIME_ID, MAYA_UFE_SEPARATOR);
-}
-
-Ufe::Path ProxyShape::ufePath() const { return Ufe::Path(ProxyShape::ufePathSegment()); }
```

Comment:
This exact method already exists in the base class (which I fixed for the |world). The ufePathSegment() method above is not needed as you can get that from this ufePath() method.